### PR TITLE
make inputs optional

### PR DESCRIPTION
**Pull request by alaamub**
## What 
Switch inputs to "not required" 

## Why
Inputs marked as required when they should not be.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -12,11 +12,15 @@ inputs:
     required: true
   resourcely_api_host:
     description: 'url for the resoucely api'
-    required: true
+    required: false
```
Comment (patrick-siewe-resourcely) [inline action.yml:15]: Why did you switch this one to optional?

Comment (alaamub) [reply action.yml]: I thought they can hard coded but i think you're right, users provide that in the action. just fixed it.

**Review by patrick-siewe-resourcely — APPROVED** [review]

**Review by drbild — APPROVED** [review]
LGTM.

File: action.yml (hunk 1 above)
Comment (drbild) [reply action.yml]: Shouldn't that be optional, with a default to `api.resourcely.io`?

**Pull request merged**